tracking/inc/trackBlock.h

```diff
@@ -0,0 +1,53 @@
+// SPDX-FileCopyrightText: 2020 CERN
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef TRACKBLOCKH
+#define TRACKBLOCKH
+
+#include "track.h"
+
+using trackBlock_t  = adept::BlockData<track>;
+
+// static std::atomic<unsigned int> atomicTrackId= 0;
+
+__host__
+void printTracks( trackBlock_t* trackBlock,       //     adept::BlockData<track>*
```

Comment:
So, the only purpose of the file is to have the print method? I would rather call it printTracks, or similar.

Reply:
I addressed this.   I renamed the file 'trackBlock.h' to 'printTracks.h'